functions/email_templates/nft_delivery.js

```diff
@@ -1,19 +1,23 @@
 function template(params) {
   // course_title, wallet_address, nft_contract, nft_id) {
   return `
-Fala builder 👷👷‍♀️!<br>
+Hello builder 👷👷‍♀️!<br>
```

Comment:
Nós estávamos deixando esses emails transacionais nas duas línguas (pt e en)

Reply:
you are right, I just updated